core/src/main/java/com/linecorp/armeria/common/NonWrappingRequestContext.java

```diff
@@ -67,18 +69,20 @@
      * Creates a new instance.
      *
      * @param sessionProtocol the {@link SessionProtocol} of the invocation
+     * @param uuid the {@link UUID} associated with this context
      * @param req the {@link HttpRequest} associated with this context
      * @param rpcReq the {@link RpcRequest} associated with this context
      */
     protected NonWrappingRequestContext(
             MeterRegistry meterRegistry, SessionProtocol sessionProtocol,
-            HttpMethod method, String path, @Nullable String query,
+            HttpMethod method, String path, UUID uuid, @Nullable String query,
```

Comment:
How about moving `uuid` right before `method`? Please make the same change for all `RequestContext` constructors you modified.